DTS-STN/Service-Canada-Labs
# Quote component accessibility

# Make quote component more accessible

Due to issues with the accessibility of the image approach to the quote component, the component has been reworked.

This PR includes:

- New component called QuoteVerticalLineContent which works with the new fragment `SCLabs-Comp-Content-v1` (a composite fragment which takes any number of separate text content fragments) and a new layout `quote`.

![Screenshot 2023-12-20 at 2 03 45 PM](https://github.com/DTS-STN/Service-Canada-Labs/assets/31868510/14d8d832-d3fc-4143-ba36-08dfb0e47df6)

## Test Instructions

1. Navigate to OAS project page
2. Navigate to newest article
3. See that quote component is working and responsive
